ext/neural_snow/NeuralSnow.jl

```diff
@@ -0,0 +1,223 @@
+module NeuralSnow
+
+using Flux
+using ClimaLand
+using ClimaLand.Snow: AbstractDensityModel, SnowModel, SnowParameters
+import ClimaLand.Snow:
+    density_prog_vars,
+    density_prog_types,
+    density_prog_names,
+    update_density!,
+    update_density_prog!
+import ClimaLand.Parameters as LP
+using Thermodynamics
+
+using DataFrames, Dates, CSV, HTTP, Flux, cuDNN, StatsBase, BSON
+ModelTools = Base.get_extension(ClimaLand, :NeuralSnowExt).ModelTools
+#^^is this correct format since this is within the extension itself, or do i just use include(./ModelTools); using ModelTools; here?
```

Comment:
I don't know the answer to this, but this seems more in line with what I do in other places:
include("./ModelTools.jl"); using ModelTools;

Reply:
e.g. here: https://github.com/CliMA/ClimaLand.jl/blob/ccc6009016f80f9ce89c42eaaf61a9fe8e9ff410/src/ClimaLand.jl#L7-L8

Reply:
sounds good to me, i'll use that version.